src/ssl.c

```diff
@@ -15693,6 +15872,17 @@ void wolfSSL_X509_STORE_CTX_set_error(WOLFSSL_X509_STORE_CTX* ctx, int err)
     (void)err;
 }
 
+#ifdef OPENSSL_EXTRA
+void wolfSSL_X509_STORE_CTX_set_time(WOLFSSL_X509_STORE_CTX* ctx,
+                                    unsigned long flags,
+                                    time_t t)
+{
+    (void)flags;
+
+    ctx->param->check_time = t; 
+    ctx->param->flags |= WOLFSSL_VERIFY_TIME;
```

Comment:
Should WOLFSSL_VERIFY_TIME be WOLFSSL_USE_CHECK_TIME to be more consistent with OpenSSL?